modular_skyrat/modules/modular_items/lewd_items/code/arousal_system.dm

```diff
@@ -11,6 +11,7 @@
 #define TRAIT_MASOCHISM		"masochism"
 #define TRAIT_SADISM		"sadism"
 #define TRAIT_NEVERBONER	"neverboner"
+#define TRAIT_BIMBO "bimbo"
```

Comment:
space it to be equal with the rest

Reply:
will do soon.

Reply:
I might want to do it in another PR where I just make a lot of sex2 code more sane.